### PR TITLE
New presence: Broken Picturephone

### PR DESCRIPTION
**Pull request by luisgabriel53**

### CODE REVIEW

File: websites/B/Broken Picturephone/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "luis ⍋",
+    "id": "403648338286870529"
+  },
+  "url": "www.brokenpicturephone.com",
+  "description": {
+    "en": "Presence to show that you're playing Broken Picturephone with details.",
+    "pt_br": "Presence que mostra que você está jogando Broken Picturephone com detalhes."
```
Comment (Bas950) [inline websites/B/Broken Picturephone/dist/metadata.json:10]: should be a description of the website not the presence

File: websites/B/Broken Picturephone/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "name": "luis ⍋",
+    "id": "403648338286870529"
+  },
+  "url": "www.brokenpicturephone.com",
+  "description": {
+    "en": "Presence to show that you're playing Broken Picturephone with details.",
+    "pt_br": "Presence que mostra que você está jogando Broken Picturephone com detalhes."
+  },
+  "service": "Broken Picturephone",
+  "version": "1.0",
```
Comment (Bas950) [inline websites/B/Broken Picturephone/dist/metadata.json:13]: version 1.0.0

File: websites/B/Broken Picturephone/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  "logo": "https://i.ibb.co/tMb7RCz/logo.png",
+  "thumbnail": "https://i.ibb.co/xq2c38y/thumbnail.png",
```
Comment (Bas950) [inline websites/B/Broken Picturephone/dist/metadata.json:15]: upload to imgur

File: websites/B/Broken Picturephone/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  "color": "#FC6807",
+  "tags": [
+    "games",
+    "game",
+    "phone",
+    "broken",
+    "broken picturephone",
+    "picturephone",
+    "telefone sem fio",
```
Comment (Bas950) [inline websites/B/Broken Picturephone/dist/metadata.json:24]: no spaces

File: websites/B/Broken Picturephone/dist/presence.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+        startTimestamp: browsingStamp
+    };
+    if (presenceData.details == null) {
+        presence.setTrayTitle();
+        presence.setActivity();
+    }
+    else {
+        presence.setActivity(presenceData);
+    }
+});
```
Comment (Bas950) [inline websites/B/Broken Picturephone/dist/presence.js:23]: Dont upload js file

File: websites/B/Broken Picturephone/presence.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+var browsingStamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", async () => {
+
+var presenceData = {
+        largeImageKey: "large_image",
+        smallImageKey: "playing",
+        smallImageText: "Playing",
+        details: "In game",
+        startTimestamp: browsingStamp
+            }
```
Comment (Bas950) [inline websites/B/Broken Picturephone/presence.ts:16]: It just shows playing and ingame please add some variation to the status depending on what you are doing

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy issues.

**Comment by fr0r** [comment]
also please send screenshots of the presence working

**Pull request closed without merging**